tests/test_tiingo_pandas.py

```diff
@@ -114,6 +114,26 @@ def test_intraday_ticker_price(self):
                                             frequency="30Min")
         self.assertGreater(len(prices), 1)
 
+    @vcr.use_cassette('tests/fixtures/ticker_price_with_volume_column.yaml')
+    def test_get_dataframe_with_volume_column(self):
+        """Confirm that requesting a single column works"""
+        requested_column = "volume"
+        prices = self._client.get_dataframe("GOOGL",
+                                               columns=requested_column,
+                                               fmt='json')
+        assert len(prices) == 1
+        assert len(prices.columns) == len(requested_column) + 1
```

Comment:
I don't think we meant to measure the length of `requested_column` for this test. That's the number of characters in the word `volume`,  not the # of columns in the dataframe.

Reply:
You are right of course. Python is not my first language, but this is some beginner stuff I missed hahaha